torch/lib/c10d/FileStore.cpp

```diff
@@ -240,7 +290,11 @@ FileStore::~FileStore() {
 void FileStore::set(const std::string& key, const std::vector<uint8_t>& value) {
   std::string regKey = regularPrefix_ + key;
   std::unique_lock<std::mutex> l(activeFileOpLock_);
+#ifdef _WIN32
+  File file(path_, O_RDWR | O_CREAT | _O_BINARY, timeout_);
```

Comment:
Couldn't we filter out this flag in File's constructor, rather than changing all the callsites?